docs/user/exercises/programming-exercise-features.inc

```diff
@@ -14,51 +14,51 @@ Instructors can still use those templates to generate programming exercises and
-  +----------------------+----------------------+----------------------+---------------------+--------------+------------------------------------------+------------------------------+----------------------------+------------------------+------------------------+
-  | Assembler            | no                   | no                   | no                  | no           | n/a                                      | no                           | no                         | yes                    | yes                    |
-  +----------------------+----------------------+----------------------+---------------------+--------------+------------------------------------------+------------------------------+----------------------------+------------------------+------------------------+
-  | Swift                | no                   | yes                  | yes                 | yes          | B: Plain, Xcode; J: Plain                | no                           | no                         | yes                    | yes                    |
-  +----------------------+----------------------+----------------------+---------------------+--------------+------------------------------------------+------------------------------+----------------------------+------------------------+------------------------+
-  | OCaml                | no                   | no                   | no                  | no           | n/a                                      | yes                          | no                         | yes                    | yes                    |
-  +----------------------+----------------------+----------------------+---------------------+--------------+------------------------------------------+------------------------------+----------------------------+------------------------+------------------------+
+  +----------------------+----------------------+----------------------+---------------------+--------------+------------------------------------------+------------------------------+----------------------------+------------------------+
+  | Programming Language | Sequential Test Runs | Static Code Analysis | Plagiarism Check    | Package Name | Project Type                             | Solution Repository Checkout | Testwise Coverage Analysis | Auxiliary repositories |
+  +======================+======================+======================+=====================+==============+==========================================+==============================+============================+========================+
+  | Java                 | B, J, L: yes; G: no  | yes                  | yes                 | yes          | B, J, L: Gradle, Maven; G: Maven         | no                           | B, J, G: yes; L: no        | B, L: yes; J, G: no    |
```

Comment:
I see that you added the LocalCI/VC support here everywhere. Did you also add the SCA and Plagiarism support for GitlabCI on purpose?

Reply:
you are right, I need to change that